seeds/create_users.js

```diff
@@ -0,0 +1,10 @@
+exports.seed = (knex) => {
+  return knex("appointments").del()
```

Comment:
Is it intentional to delete all appointments here?

Reply:
Yeah but if we want the seed to just add more every time, I can remove it.

Reply:
no that's fair, if the idea for seed is to restore the db to the same state then this makes sense 👍